editor/property_editor.cpp

```diff
@@ -3642,8 +3646,12 @@ void PropertyEditor::update_tree() {
 						item->set_tooltip(1, res->get_name() + " (" + res->get_class() + ")");
 					}
 					if (!res->is_class("Texture")) {
-						//texture already previews via itself
-						EditorResourcePreview::get_singleton()->queue_edited_resource_preview(res, this, "_resource_preview_done", item->get_instance_ID());
+						if (res->is_class("Script")) {
+							item->set_text(1, res->get_path().get_file());
+						} else {
+							//texture already previews via itself
```

Comment:
@bojidar-bg 
There are several `item->set_text(1, ...)` on upper lines.
I was afraid to make any side effect (bug) with what you suggested `if else` code.
I'm not 100% sure it's safe.